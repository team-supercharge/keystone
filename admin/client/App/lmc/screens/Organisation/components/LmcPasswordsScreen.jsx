import React, { Component } from 'react'
import PropTypes from 'prop-types'
import { connect } from 'react-redux'

export class LmcPasswordsScreen extends Component {
    render() {
        return (
            <div style={{ padding: 50 }}>
                <table cellPadding="0" cellSpacing="0" className="Table ItemList">
					<thead>
                        <tr>
                            <th>
                                Title
                            </th>
                            <th>
                                Start
                            </th>
                            <th>
                                End
                            </th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>
                                Shift 1
                            </td>
                            <td>
                                08:99
                            </td>
                            <td>
                                13:22
                            </td>
                        </tr>
                    </tbody>
				</table>
            </div>
        )
    }
}

LmcPasswordsScreen.propTypes = {}

const mapStateToProps = state => {
    return {}
}

const mapDispatchToProps = dispatch => {
    return {}
}

export default connect(mapStateToProps, mapDispatchToProps)(LmcPasswordsScreen)