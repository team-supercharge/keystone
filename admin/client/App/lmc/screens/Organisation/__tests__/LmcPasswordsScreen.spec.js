import React from 'react'
import { shallow } from 'enzyme'
import { LmcPasswordsScreen } from '../components/LmcPasswordsScreen.jsx'

describe('LmcPasswordsScreen', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallow(
            <LmcPasswordsScreen />
        )
    })

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot()
    })
})