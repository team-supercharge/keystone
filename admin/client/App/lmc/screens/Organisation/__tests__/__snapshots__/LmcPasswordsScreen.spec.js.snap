// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LmcPasswordsScreen renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    style={
      Object {
        "padding": 50,
      }
    }
  >
    <table
      cellPadding="0"
      cellSpacing="0"
      className="Table ItemList"
    >
      <thead>
        <tr>
          <th>
            Title
          </th>
          <th>
            Start
          </th>
          <th>
            End
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            Shift 1
          </td>
          <td>
            08:99
          </td>
          <td>
            13:22
          </td>
        </tr>
      </tbody>
    </table>
  </div>,
  "nodes": Array [
    <div
      style={
        Object {
          "padding": 50,
        }
      }
    >
      <table
        cellPadding="0"
        cellSpacing="0"
        className="Table ItemList"
      >
        <thead>
          <tr>
            <th>
              Title
            </th>
            <th>
              Start
            </th>
            <th>
              End
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Shift 1
            </td>
            <td>
              08:99
            </td>
            <td>
              13:22
            </td>
          </tr>
        </tbody>
      </table>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <LmcPasswordsScreen />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": LmcPasswordsScreen {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          style={
            Object {
              "padding": 50,
            }
          }
        >
          <table
            cellPadding="0"
            cellSpacing="0"
            className="Table ItemList"
          >
            <thead>
              <tr>
                <th>
                  Title
                </th>
                <th>
                  Start
                </th>
                <th>
                  End
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  Shift 1
                </td>
                <td>
                  08:99
                </td>
                <td>
                  13:22
                </td>
              </tr>
            </tbody>
          </table>
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          style={
            Object {
              "padding": 50,
            }
          }
        >
          <table
            cellPadding="0"
            cellSpacing="0"
            className="Table ItemList"
          >
            <thead>
              <tr>
                <th>
                  Title
                </th>
                <th>
                  Start
                </th>
                <th>
                  End
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  Shift 1
                </td>
                <td>
                  08:99
                </td>
                <td>
                  13:22
                </td>
              </tr>
            </tbody>
          </table>
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <LmcPasswordsScreen />,
}
`;
